### PR TITLE
fix(ui5-color-palette): focus outline is properly visualized

### PR DESCRIPTION
**Pull request by unazko**
- The color palette swatches focus outline is properly visualized
according to the global focus visual concept.

Fixes: #3521

### CODE REVIEW

**Comment by tsanislavgatev** [comment]
With removing of the styles we now receive a blue outline which is the browser provided focus. We need this to prevent it:
:host(:not([_disabled])) .ui5-cp-item:focus{
	outline: none;
}

:host(:not([_disabled]):focus) .ui5-cp-item {
	pointer-events: none;
	outline: none;
}

File: packages/main/src/themes/ColorPaletteItem.css [hunk 1]
```diff
@@ -7,24 +7,46 @@
...
-	margin: var(--_ui5_color-palette-item-hover-margin);
+:host(:not([_disabled])) .ui5-cp-item:focus::before {
+	content: "";
+	box-sizing: border-box;
+	position: absolute;
+	left: 0.125rem;
+	top: 0.125rem;
+	right: 0.125rem;
+	bottom: 0.125rem;
+	border: 0.0625rem solid black;
```
Comment (fifoosid) [inline packages/main/src/themes/ColorPaletteItem.css:38]: white

File: packages/main/src/themes/ColorPaletteItem.css [hunk 2]
```diff
@@ -7,24 +7,46 @@
...
-    height: 100%;
+:host(:not([_disabled])) .ui5-cp-item:focus::after {
+	content: "";
+	box-sizing: border-box;
+	position: absolute;
+	left: 0.125rem;
+	top: 0.125rem;
+	right: 0.125rem;
+	bottom: 0.125rem;
+	border: 0.0625rem dotted white;
```
Comment (fifoosid) [inline packages/main/src/themes/ColorPaletteItem.css:50]: black

**Review by fifoosid — CHANGES_REQUESTED** [review]
Just switch the two colours and we can merge it.
Reason:
Current result:
image

Expected behaviour:
image

File: packages/main/src/themes/ColorPaletteItem.css (hunk 1 above)
Comment (unazko) [reply packages/main/src/themes/ColorPaletteItem.css]: done

File: packages/main/src/themes/ColorPaletteItem.css (hunk 2 above)
Comment (unazko) [reply packages/main/src/themes/ColorPaletteItem.css]: done

**Review by fifoosid — APPROVED** [review]

**Pull request merged**